src/Ant/Modal.elm

```diff
@@ -487,12 +500,19 @@ toHtml isVisible (Modal opts contents) =
                     attr "2-row"
 
         modalHtml =
-            H.div [ class modalContainerClass, layoutInfoAttr ]
+            S.div
+                [ A.class modalContainerClass
+                , layoutInfoAttr
+                , css
+                    [ top (pct opts.verticalOffset)
```

Comment:
See related comment above. Would be nice to have a union type to allow for either percent or pixel offsets.